library/core/src/macros/mod.rs

```diff
@@ -334,16 +334,16 @@ macro_rules! r#try {
 /// formatted according to the specified format string and the result will be passed to the writer.
 /// The writer may be any value with a `write_fmt` method; generally this comes from an
 /// implementation of either the [`std::fmt::Write`] or the [`std::io::Write`] trait. The macro
-/// returns whatever the `write_fmt` method returns; commonly a [`std::fmt::Result`], or an
-/// [`io::Result`].
+/// returns whatever the `write_fmt` method returns; commonly a [`std::fmt::Result`], or a
+/// [`std::io::Result`].
```

Comment:
I liked it better before actually - it's idiomatically used as `io::Result`, so it makes sense to refer to it that way.

Reply:
Maybe change the other one to `fmt::Result` then to match it?

```suggestion
/// returns whatever the `write_fmt` method returns; commonly a [`fmt::Result`], or a
/// [`io::Result`].
```